app/src/main/kotlin/compiler/intermediate_form/AsmAble.kt

```diff
@@ -0,0 +1,3 @@
+package compiler.intermediate_form
+
+sealed interface AsmAble
```

Comment:
`Asmable` (this is supposed to be a single adjective)

Reply:
True